From f76c05997bf6b85ecfe529137b4e54794460779a Mon Sep 17 00:00:00 2001
From: Hu hui <b29976@freescale.com>
Date: Sat, 9 Oct 2010 11:28:34 +0800
Subject: [PATCH] ENGR00132417 IMX53 USB  fix build error for 53 ard board

fix build error for mx53 ARD board with USB HOST2

Signed-off-by: Hu Hui <b29976@freescale.com>
---
 arch/arm/plat-mxc/usb_common.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/plat-mxc/usb_common.c b/arch/arm/plat-mxc/usb_common.c
index 5a60f06..1f6a902 100644
--- a/arch/arm/plat-mxc/usb_common.c
+++ b/arch/arm/plat-mxc/usb_common.c
@@ -346,7 +346,7 @@ static void usbh2_set_ulpi_xcvr(void)
 	USBCTRL_HOST2 &= ~UCTRL_H2WIE;	/* wakeup intr enable */
 	USBCTRL_HOST2 &= ~UCTRL_H2UIE;	/* ULPI intr enable */
 	USB_CTRL_1 |= USB_CTRL_UH2_EXT_CLK_EN;
-	if (cpu_is mx53())
+	if (cpu_is_mx53())
 		USB_CTRL_1 |= USB_CTRL_UH2_CLK_FROM_ULPI_PHY;
 	if (cpu_is_mx51())/* not tested */
 		USBCTRL_HOST2 |= (1 << 12);
-- 
1.5.4.4

